ext/opentelemetry-ext-prometheus/README.rst

```diff
@@ -0,0 +1,71 @@
+OpenTelemetry Prometheus Exporter
+=============================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-ext-prometheus.svg
+   :target: https://pypi.org/project/opentelemetry-ext-prometheus/
+
+This library allows to export metrics data to `Prometheus <https://prometheus.io/>`_.
+
+Installation
+------------
+
+::
+
+     pip install opentelemetry-ext-prometheus
+
+
+Usage
+-----
+
+The **OpenTelemetry Prometheus Exporter** allows to export `OpenTelemetry`_ metrics to `Prometheus`_.
+
+
+.. _Prometheus: https://prometheus.io/
+.. _OpenTelemetry: https://github.com/open-telemetry/opentelemetry-python/
+
+.. code:: python
+
+    from opentelemetry import metrics
```

Comment:
It'd be nice to have this example as a Python file, so one can execute it without copy-paste.